xml/System.Reflection.Metadata.Ecma335/MetadataBuilder.xml

```diff
@@ -740,15 +846,25 @@
         <Parameter Name="length" Type="System.Int32" Index="5" FrameworkAlternate="netcore-1.1;netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="method">To be added.</param>
-        <param name="importScope">To be added.</param>
-        <param name="variableList">To be added.</param>
-        <param name="constantList">To be added.</param>
-        <param name="startOffset">To be added.</param>
-        <param name="length">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="method">The containing method.</param>
+        <param name="importScope">The handle of the associated import scope.</param>
+        <param name="variableList">If the scope declares variables, set this to the handle of the first one. Otherwise, set this to the handle of the first variable declared by the next scope definition. If no scope defines any variables, <see cref="M:System.Reflection.Metadata.Ecma335.MetadataTokens.LocalVariableHandle(System.Int32)" />(1).</param>
```

Comment:
```suggestion
        <param name="variableList">If the scope declares variables, set this to the handle of the first one. Otherwise, set this to the handle of the first variable declared by the next scope definition. If no scope defines any variables, <see langword="System.Reflection.Metadata.Ecma335.MetadataTokens.LocalVariableHandle(1)" />.</param>
```